app/models/tasks.rb

```diff
@@ -0,0 +1,59 @@
+module Tasks
```

Comment:
i think in the future we need to push these down even further. ie every task gets its own class (decorator thing?) we can delegate to. at the moment they kind of overlap and not so great to work with

Reply:
Yeah I think tasks are pretty tricky to work with. We should probably have a wider discussion around how tasks work. Having incomplete tasks not be a thing and being represented by a string and then only having a record created for them when they're passed / failed isn't ideal, not to mention the whole `claim_verifier_match` stuff